script_library/deployment-actions-openstack.sh

```diff
@@ -89,6 +94,8 @@ function clean_openstack(){
     run_ansible ${socok8s_absolute_dir}/playbooks/openstack-deploy_caasp.yml -e caasp_stack_delete=True || true
     echo "Delete SES node"
     run_ansible ${socok8s_absolute_dir}/playbooks/openstack-ses_aio_instance.yml -e ses_node_delete=True
+    echo "Delete network stack"
+    run_ansible ${socok8s_absolute_dir}/playbooks/openstack-cleanup_network.yml
```

Comment:
This will actively clean things in gates, as we are running this function. So it means the gates will remove the network, and not recreate it. That's not ideal.

Reply:
Then I would prefer to create networks in the default steps instead of not cleaning up networks. I don't see a reason why we should treat networks different to any other resources.

Reply:
I guess that's where I differ: Networks don't need to be unique. We could all share the same networks. I don't see a reason to isolate those, because it takes longer to build things in CI for no positive value.

Reply:
- It doesn't take that much longer (this is in the range of tens of seconds I would say) to create a new network
- having isolated networks is nice because you know that only your deployed services are on this network (and no other dhcp/pxe/etcd/whatever service is running beside the one you want to have)

Or is your plan to run multiple envs in the same network?

Reply:
> I guess that's where I differ: Networks don't need to be unique. We could all share the same networks

While it might be possible. I really think it's a bad idea. Especially when done just to safe ~10-15s of deployment time. We should have different deployment isolated networkwise.

Reply:
Current code creates a network stack if the stack is not already available. @evrardjp @rhafer is that ok for now?

Reply:
It's okay.